gtsfm/utils/verification.py

```diff
@@ -75,37 +80,48 @@ def essential_to_fundamental_matrix(
 
 
 def compute_epipolar_distances(
-    normalized_coords_i1: np.ndarray, normalized_coords_i2: np.ndarray, i2Ei1: EssentialMatrix
+    coordinates_i1: np.ndarray, coordinates_i2: np.ndarray, i2Fi1: np.ndarray, distance_type: str = "sed"
 ) -> Optional[np.ndarray]:
-    """Compute symmetric point-line epipolar distances between normalized coordinates of correspondences.
+    """Compute point-line epipolar distance between corresponding coordinates in two images.
 
     Args:
-        normalized_coords_i1: normalized coordinates in image i1, of shape Nx2.
-        normalized_coords_i2: corr. normalized coordinates in image i2, of shape Nx2.
-        i2Ei1: essential matrix between two images
+        coordinates_i1: coordinates in image i1, of shape Nx2.
+        coordinates_i2: corr. coordinates in image i2, of shape Nx2.
+        i2Fi1: fundamental matrix between two images.
+        distance_type (optional): type of distance metric to compute. The options are "sed" (symmetric epipolar
```

Comment:
we need a unit test for both the distance types. Currently it is there for just one. @jingwuOUO  can you extend it?